site/kube-addon.md

```diff
@@ -210,7 +278,7 @@ TCP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 ```
 
-### <a name="configuration-options"></a> Changing Configuration Options
+## <a name="configuration-options"></a> Changing Configuration Options
 
 #### Using `cloud.weave.works`
 
```

Comment:
### Launching the Weave Net Agent in Weave Cloud

Should also link to the install section in Weave Cloud or I should add this there too... *shrug* It's best to keep the install parts for Weave cloud in one place, IMO.  I could link here from WC too.